packages/ui-extensions/src/docs/shared/components/Heading.ts

```diff
@@ -3,7 +3,7 @@ import type {SharedReferenceEntityTemplateSchema} from '../docs-type';
 const data: SharedReferenceEntityTemplateSchema = {
   name: 'Heading',
   description:
-    'The `s-heading` component renders hierarchical titles similar to HTML’s `h1-h6` elements. Heading levels are automatically determined by parent `s-section` nesting depth starting from `h2`. Always prefer using the `heading` property on `s-section` and `s-banner` components before constructing custom layout with `s-heading`.',
+    'Renders hierarchical titles to communicate the structure and organization of page content. Heading levels adjust automatically based on nesting within parent Section components, ensuring a meaningful and accessible page outline.',
```

Comment:
```suggestion
    'Renders hierarchical titles to communicate the structure and organization of page content. Heading levels adjust automatically based on nesting within parent `Section` components, ensuring a meaningful and accessible page outline.',
```
? or are we doing this only when mentioning the custom element ?

Reply:
Yeah, I left these like this intentionally. They are not referencing the code, just the component's name.

Reply:
It aligns with the comment from Sam below that I saw after answering here.